### PR TITLE
Metrics module

### PR DESCRIPTION
**Pull request by lukasz-zimnoch**
<s>Depends on keep-network/keep-common#40</s>
<s>Depends on https://github.com/keep-network/keep-core/pull/1850</s>

Here we introduce some basic metrics gathering and exposing them through an HTTP endpoint.

**Metrics table**
| Metric | Description |
| ------ | ------------ |
| connected_peers_count | Presents the count of all connected peers |
| connected_bootstrap_count | Presents the count of connected bootstraps from the list of bootstraps defined in the configuration |
| eth_connectivity | Presents whether the connection with ETH node is up (1 - yes, 0 - no) |
| libp2p_info | Presents some libp2p specific data |

**Example response from the `/metrics` endpoint**

Presented output follows the [Prometheus text-based exposition format](https://prometheus.io/docs/instrumenting/exposition_formats/)

```
# TYPE connected_peers_count gauge
connected_peers_count 62 1592993341396

# TYPE connected_bootstrap_count gauge
connected_bootstrap_count 10 1592993341396

# TYPE eth_connectivity gauge
eth_connectivity 1 1592993341421

libp2p_info{id="16Uiu2HAkuTUKNh6HkfvWBEkftZbqZHPHi3Kak5ZUygAxvsdQ2UgG"} 1
```

### CODE REVIEW

**Comment by pdyraga** [comment]
@lukasz-zimnoch Can you please summarize metrics in the PR description, just like you did for `keep-core`?

File: cmd/start.go [hunk 1]
```diff
@@ -173,3 +181,53 @@ func Start(c *cli.Context) error {
 		return fmt.Errorf("unexpected context cancellation")
 	}
 }
+
+func initializeMetrics(
+	ctx context.Context,
+	config *config.Config,
+	netProvider net.Provider,
+	stakeMonitor chain.StakeMonitor,
+) error {
```
Comment (pdyraga) [inline cmd/start.go:190]: Is it possible we'll ever get `error` from here? I think I missed it when reviewing https://github.com/keep-network/keep-core/pull/1850/

Comment (lukasz-zimnoch) [reply cmd/start.go:190]: Yeah, currently not.

Comment (pdyraga) [reply cmd/start.go:190]: Can we drop this `error` and do a quick PR for `keep-core` as well?

Comment (lukasz-zimnoch) [reply cmd/start.go]: Sure

**Comment by pdyraga** [comment]
```
➜  ~ curl -D - http://localhost:8081/metrics
HTTP/1.1 200 OK
Date: Fri, 03 Jul 2020 14:16:28 GMT
Content-Length: 293
Content-Type: text/plain; charset=utf-8

# TYPE connected_peers_count gauge
connected_peers_count 2 1593785768629

# TYPE connected_bootstrap_count gauge
connected_bootstrap_count 2 1593785768629

# TYPE eth_connectivity gauge
eth_connectivity 0 1593785768629

libp2p_info{id="16Uiu2HAm2TUrDJbpAyuVwhNRmdju6QyQG1Q4661CvCKNgtN9Z6NQ"}
```

works as advertised :+1:

**Pull request merged**